### PR TITLE
Updated Background

### PR DESCRIPTION
**Pull request by VasaviLagishetty**
background

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -4,11 +4,11 @@
 <head>
   {% include force-https.html %}
   <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
```
Comment (SriSatyaLokesh) [inline _includes/head.html]: you need to keep this content variables.

may be you accidentally removed them.

add back if there's no problem

File: _includes/head.html [hunk 2]
```diff
@@ -37,7 +37,7 @@
   {% endif %}
 
   {% if page.url == "/404.html" | prepend: {{site.baseurl}} %}
-    <meta http-equiv="refresh" content="20; url={{site.baseurl}}/">
+    <meta http-equiv="refresh" ="20; url={{site.baseurl}}/">
```
Comment (SriSatyaLokesh) [inline _includes/head.html]: even here 
without var name `content` it's completely in accurate

File: css/grayscale.scss [hunk 3]
```diff
@@ -495,3 +495,64 @@ body {
...
+  }
+  
+  @keyframes animStar {
```
Comment (SriSatyaLokesh) [inline css/grayscale.scss:550]: i tried the same but failed for me.

**Review by SriSatyaLokesh — CHANGES_REQUESTED** [review]
Great job

Please look at those comments and make changes

File: _includes/head.html (hunk 1 above)
Comment (VasaviLagishetty) [reply _includes/head.html]: yeah I will look into it

**Review by SriSatyaLokesh — APPROVED** [review]

**Pull request merged**